tests/test_directives.py

```diff
@@ -144,42 +152,42 @@ class TestDirectives:
     """
     Tests for Directives Class
     """
-    def test_get_directive(self, available_directives_list):
-        directives = Directives(available_directives_list)
+    def test_get_directive(self, directives_fixture, available_directives_list):
+        directives = directives_fixture()
         for item in available_directives_list:
             assert directives[item.name] == item.default
 
-    def test_add_directive(self, available_directives_list):
-        directives = Directives(available_directives_list[:-1])
+    def test_add_directive(self, directives_fixture, available_directives_list):
+        directives = directives_fixture(-1)
```

Comment:
Given how we use `directives_fixture` I think it makes sense just to return the directives item. I would then just manually create it here, since it is the only place we put in a value into the fixture function.